CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## 1.0.7 – 2024-03-22
+
+### Added
+
+- Custom input form for Scoped Context Chat @kyteinsky [#45](https://github.com/nextcloud/assistant/pull/45)
```

Comment:
Technically this would be 1.1.0 then ;) https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/release_process.html#minor-updates

#semver